lndr-backend/src/Lndr/Config.hs

```diff
@@ -0,0 +1,40 @@
+{-# LANGUAGE OverloadedStrings         #-}
+
+module Lndr.Config where
+
+import           Data.Configurator
+import           Data.Configurator.Types
+import qualified Data.HashMap.Strict     as H (lookup)
+import           Data.Maybe                 (fromMaybe)
+import qualified Data.Text               as T
+import           Lndr.Types
+import           System.Environment      (setEnv)
+import           System.FilePath
+
+loadConfig :: IO ServerConfig
+loadConfig = do
+    config <- getMap =<< load [Required $ "lndr-backend" </> "data" </> "lndr-server.config"]
+    let loadEntry x = fromMaybe (error $ T.unpack x) $ convert =<< H.lookup x config
+    return $ ServerConfig (loadEntry "lndrUcacAddr")
+                          (loadEntry "creditProtocolAddress")
+                          (loadEntry "issueCreditEvent")
+                          (loadEntry "scanStartBlock")
+                          (loadEntry "dbUser")
+                          (loadEntry "dbUserPassword")
+                          (loadEntry "dbName")
+                          (loadEntry "executionAddress")
+                          (loadEntry "gasPrice")
+                          (loadEntry "maxGas")
+                          (loadEntry "urbanAirshipKey")
+                          (loadEntry "urbanAirshipSecret")
+                          (loadEntry "heartbeatInterval")
+                          (loadEntry "awsPhotoBucket")
+                          (loadEntry "accessKeyId")
+                          (loadEntry "secretAccessKey")
+                          (loadEntry "postgresHost")
+                          (loadEntry "postgresPort")
```

Comment:
**Non-blocking:** We're using the `db` prefix above for database-related configuration, but here we're using a `postgres` prefix. Should both sets of configuration values use the same prefix?

Reply:
right, this was sloppy!